arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -943,6 +899,18 @@
 				<key>descriptionText</key>
 				<string>Use different feature request modes to populate the map from a service feature table.</string>
 			</dict>
+			<dict>
```

Comment:
I see you mentioned at the top of this PR, in the discuss session. If we decide to move it to the top of the category table view, this entry in the plist needs to go to the front of the array.

Reply:
I was thinking I'd reorganize after we decide how we want to reorder the list. 

@philium How do you feel about reorganizing the order of samples? Possibly alphabetically or by complexity?

Reply:
We may need to defer until further discussion, see `/common-samples/issues/3266` and #810 - make sure M&T know the concerns for ordering alphabetically. 👀

Reply:
I am all for sorting the samples and I think it makes sense to do so alphabetically by name.